Plugin/src/main/java/dev/rosewood/rosestacker/manager/StackManager.java

```diff
@@ -70,6 +71,30 @@ public void reload() {
             long interval = autosaveFrequency * 20 * 60;
             this.autosaveTask = Bukkit.getScheduler().runTaskTimer(this.rosePlugin, () -> this.saveAllData(false), interval, interval);
         }
+
+        this.stackingBasedOnPerformance = ConfigurationManager.Setting.PERFORMANCE_TPS_TOGGLE.getBoolean();
+        if(this.stackingBasedOnPerformance){
+            Bukkit.getServer().getScheduler().runTaskAsynchronously(rosePlugin, new Runnable() {
+                @Override
+                public void run() {
+                    if(ConfigurationManager.Setting.PERFORMANCE_TPS_TOGGLE.getBoolean()){
```

Comment:
Not sure why this is being checked again? You checked it above and had a variable assigned to it, but aren't using that variable here.